src/Compilers/Core/Portable/EmbeddedText.cs

```diff
@@ -10,7 +10,6 @@
 using System.Reflection.Metadata;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Debugging;
```

Comment:
Consider reverting changes in this file.

Reply:
The import is no longer needed as the namespace of SourceHashAlgorithms changed to Microsoft.CodeAnalysis.Text.